cli/pcluster/config/mappings.py

```diff
@@ -477,6 +478,12 @@
         "s3_read_write_resource": {
             "cfn_param_mapping": "S3ReadWriteResource",  # TODO add validator
         },
+        "disable_hyperthreading": {
+            "type": DisableHyperThreading,
+            "default": "false",
+            "allowed_values": ["true", "false"],
```

Comment:
The new parameter is a boolean so you should define the new param type by inheriting the `BoolParam` type :
` class DisableHyperThreading(BoolParam):  `

Then you don't need to specify the `allowed_values` and the `default` can be set to `False`.

```python
 "disable_hyperthreading": {
     "type": DisableHyperThreadingParam,
     "default": False,
     "cfn_param_mapping": "Cores",
}
```